### PR TITLE
Space Station 13: Holy Furry part 1

### PR DESCRIPTION
**Pull request by ghost**
### Intent of your Pull Request

Deus vult!

The holy book now can convert more cult items, and the cult soulshard now doesn't delete itself.

Cultblade into holy claymore
Cult Hardsuit into inquisition hardsuit
Cult normal armor into riot chaplain armor 
Cult hood into chaplain helmet
Cult blade into null rod
Cult soulshard now is reusable

#### Changelog

:cl:  
rscadd: The holy book can convert more cult items, such as cult hardsuits, armor, blades, et cetera!
/:cl:

### CODE REVIEW

**Comment by boodaliboo** [comment]
your going ham today good job loving these

File: code/game/objects/items/storage/book.dm [hunk 1]
```diff
@@ -198,7 +198,68 @@ GLOBAL_LIST_INIT(bibleitemstates, list("bible", "koran", "scrapbook", "burning",
 			new /obj/item/nullrod/claymore(get_turf(sword))
 			user.visible_message("<span class='notice'>[user] has purified the [sword]!</span>")
 			qdel(sword)
-
+	//yogs start -- More Cult conversions!//
+	else if(istype(A, /obj/item/melee/cultblade) && !iscultist(user))
```
Comment (Altoids1) [inline code/game/objects/items/storage/book.dm:202]: Can you shunt this long else-if chain into a proc of ``/obj/item/storage/book/bible``? Like
```dm
	//yogs start -- other conversions
	else if(other_conversions())
		//a null if-statement with all the action happening in the proc
	//yogs end
	else if(istype(A, /obj/item/soulstone) && !iscultist(user))
```
Or something?

**Review by Altoids1 — CHANGES_REQUESTED** [review]

**Comment by HomocidlePancakes** [comment]
Oh god. Chap will massproduce nullrods now. No way this can end good

**Comment by HomocidlePancakes** [comment]
Oh wait. EVERYONE can purify. This is fucked

**Pull request closed without merging**